src/components/AddQuote.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import './AddQuote.css'
+
+class AddQuote extends React.Component {
+    constructor(props){
+        super(props);
+        this.state={
+            autor:'',
+            quote:''
+        }
+        this.addQuote=this.addQuote.bind(this);
+        this.handleInput=this.handleInput.bind(this);
+        this.deleteQuote=this.deleteQuote.bind(this);
+    }
+    
+  handleInput(event){
+    const target = event.target;
+    const value = target.value;
+    const name = target.name;
+    this.setState({[name]: value});
+  }
+    
+  addQuote(e){
+      e.preventDefault();
+      fetch("https://baza-podataka.herokuapp.com/dodaj-citat/",
```

Comment:
trebalo bi da se napravi neki config provider , npr po 12factor metodologiji

Reply:
evo dodato sam karticu za to https://trello.com/c/lO5vhsMz/31-dodati-konfiguraciju-projekta